src/ConstructiveSolidGeometry/SurfacePrimitives/Polygon.jl

```diff
@@ -0,0 +1,99 @@
+
+struct Polygon{N,T} <: AbstractPlanarSurfacePrimitive{T}
+    points::SVector{N, CartesianPoint{T}} 
+end
+const Triangle{T} = Polygon{3, T}
+const Quadrangle{T} = Polygon{4, T}
+
+normal(p::Polygon) = normalize((p.points[2] - p.points[1]) × (p.points[3] - p.points[1]))
```

Comment:
Does this always have the right orientation?

Reply:
I think so :thinking:  
Plots of Polygons with their normal vectors give the "correct" direction of the normal.